elrond-wasm-derive/src/parse/supertrait_parse.rs

```diff
@@ -1,5 +1,18 @@
 use crate::{model::Supertrait, parse::split_path_last};
 
+pub fn is_contract_base(supertrait: &syn::TypeParamBound) -> bool {
+    match supertrait {
+        syn::TypeParamBound::Trait(t) => {
+            if let Ok(path) = syn::parse_str::<syn::Path>("elrond_wasm::contract_base::ContractBase") {
```

Comment:
We can do better than this, this limits how `ContractBase` can be specified. It is possible to break the Path into components and only look at the last component.

We are already doing this Path parsing somewhere. This condition can be expressed as "has a module with name ContractBase".

Reply:
Good point I totally forgot it! Done